This file contains a patch to the Linux file /usr/share/magic which will
allow the "file" command to properly identify tags file producted by
Exuberant Ctags. To apply the patch, issue the following command as root:

    patch -p0 < magic.diff

*** /usr/share/magic.orig	Wed Feb 16 19:04:09 2000
--- /usr/share/magic	Mon Aug 14 20:27:01 2000
***************
*** 1155,1160 ****
--- 1155,1164 ----
  >84	belong&0x18000000	=0x18000000	undefined fpmode
  
  #------------------------------------------------------------------------------
+ # ctags:  file (1) magic for Exuberant Ctags files
+ 0	string	!_TAG	Exuberant Ctags tag file
+ 
+ #------------------------------------------------------------------------------
  # database:  file(1) magic for various databases
  #
  # extracted from header/code files by Graeme Wilford (eep2gw@ee.surrey.ac.uk)
